### PR TITLE
s390x: Add workaround for failing yast2_lan tests

### PR DESCRIPTION
**Pull request by Soulofdestiny**

### CODE REVIEW

**Comment by Soulofdestiny** [comment]
verification run: opeth.suse.de/tests/4700

File: tests/console/yast2_lan.pm [hunk 1]
```diff
@@ -28,6 +28,13 @@ sub handle_Networkmanager_controlled {
     wait_serial("yast2-lan-status-0", 60) || die "'yast2 lan' didn't finish";
 }
 
+sub handle_dhcp_popup {
+    if (check_var('ARCH', 's390x') && match_has_tag('dhcp-popup')) {
+        send_key 'alt-o';
+        wait_still_screen(3);
```
Comment (okurz) [inline tests/console/yast2_lan.pm]: because it is an ncurses console test I think that should be

```perl
wait_screen_change { send_key 'alt-o' };
```

**Comment by Soulofdestiny** [comment]
updated, new verification run: opeth.suse.de/tests/4701  @okurz

File: products/sle/main.pm [hunk 2]
```diff
@@ -1306,9 +1310,9 @@ else {
         }
     }
     unless (load_applicationstests() || load_slenkins_tests()) {
-        load_rescuecd_tests();
-        load_consoletests();
-        load_x11tests();
+        # load_rescuecd_tests();
+        # load_consoletests();
+        # load_x11tests();
```
Comment (okurz) [inline products/sle/main.pm]: no

Comment (okurz) [inline products/sle/main.pm]: fix this and merge

Comment (Soulofdestiny) [inline products/sle/main.pm]: yeah.. just noticed...

**Pull request merged**